src/docs/tests.rs

```diff
@@ -41,19 +43,19 @@ pub fn complicated_fun(
 
     let config = PackageConfig {
         name: "test".to_string(),
-        docs: Default::default(),
-        tool: Default::default(),
-        version: Default::default(),
-        repository: Default::default(),
-        description: Default::default(),
-        dependencies: Default::default(),
+        docs: crate::config::Docs::default(),
+        tool: crate::config::BuildTool::default(),
+        version: String::default(),
+        repository: crate::config::Repository::default(),
+        description: String::default(),
+        dependencies: std::collections::HashMap::default(),
```

Comment:
This seems worse, what is the reason for this one? The Rust type checker can easily infer this information.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#default_trait_access
> It’s more clear to the reader to use the name of the type whose default is being gotten than the generic Default.